src/Troubleshooting/index.md

```diff
@@ -24,38 +21,25 @@ Run `echo kernel.split_lock_mitigate=0 | sudo tee /etc/sysctl.d/99-splitlock.con
 
 ---
 
-## General Issues
-
-Known Vinegar issues which affect both Roblox Player and Roblox Studio:
-
-### Wine is not supported
-As the name suggests, Wine support in the Roblox Player has been disabled.
-For further information, read [here.](https://vinegarhq.org/Home/rol_faq.html)
-
-Workarounds such as running Roblox in a virtual machine or Android emulation with Waydroid are still possible; read the pinned thread in the helpdesk channel present in the VinegarHQ discord server for more information.
+## Issues
 
 ### Cursor fails to lock
 
-On XWayland, the cursor cannot be locked unless it's invisible. This impacts the functionality of Roblox's first-person and shiftlock modes. This also impacts Studio's edit mode, where the camera rotation depends on the cursor being locked and is broken.
-
-Switching to a real X11 session fixes this issue. Some users have also reported success with gamescope's `--force-grab-cursor` option, but this doesn't seem to work for everyone.
+On XWayland, the cursor cannot be locked unless it's invisible, this shouldn't be an issue on Flatpak.
 
-Once Wine's wayland driver is introduced (scheduled for next year), this issue should be fixed.
+Switching to a real X11 session fixes this issue.
 
-### DXVK returns white screen
-
-As of 16/02/24 Roblox released an update that broke DXVK,
-a temporary solution is to disable DXVK and use Vulkan instead.
+Using the [Kron4ek Proton WINE builds](https://github.com/Kron4ek/Wine-Builds/releases) will fix this.
 
 ### White Screen
 
-Usually, this is a sign of missing graphics libraries which Wine depends on to work. Check [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) out for instructions on how to install them.
+Usually, this is a sign of missing graphics libraries which Wine depends on to work (This doesn't apply to Flatpak) check [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) out for instructions on how to install them.
```

Comment:
Dont add "this doesn't apply to flatpak", since this assumes the user will reach this problem within the flatpak, and cause them to get stumped, and flip the order (is your GPU supported? then install drivers...)